CRM/Event/Form/Participant.php

```diff
@@ -1670,15 +1670,6 @@ public function buildEventFeeForm($form) {
       $form->addElement('hidden', 'hidden_eventFullMsg', $eventfullMsg, ['id' => 'hidden_eventFullMsg']);
     }
 
-    if ($form->_pId) {
-      if (CRM_Core_DAO::getFieldValue('CRM_Event_DAO_ParticipantPayment',
-        $form->_pId, 'contribution_id', 'participant_id'
-      )
-      ) {
-        $form->_online = !$form->isBackOffice;
```

Comment:
$form->_online is still used a bit lower down on line 1711 - or is that what you mean by it's giving E_NOTICES?

Reply:
@demeritcowboy no - it's the $form->_pId that is but that drew me here. I think $form->_online could only ever be/ remain false because $form->isBackOffice can only be true

Reply:
@demeritcowboy I removed the second reference too